packages/streamline/src/components/list-items/list-item-selectable/components/left-icon.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box } from '../../../../primitives/box/box';
+import { Icon } from '../../../../primitives/icon/icon';
+import { ColorTheme } from '../../../../theme';
+import Spinner from '../../../spinner/spinner';
+
+export const LeftIcon = ({
+  isSelected = false,
+  isLoading,
+  color,
+  testID,
+}: {
+  isSelected?: boolean;
+  color: ColorTheme;
+  isLoading: boolean;
+  testID?: string;
+}) => {
+  if (isLoading)
+    return (
+      <Box marginRight="md">
+        <Spinner color={color} size="regular" />
+
+        <Box width={24} />
+      </Box>
+    );
+
+  return (
+    <Box marginRight="md">
+      {isSelected ? (
+        <Icon
+          color={color}
+          iconName="Check"
+          size="large"
+          testID={`${testID}-check-icon`}
+        />
+      ) : (
+        <Box width={24}></Box>
```

Comment:
Shouldn't it be handled by increasing the `marginRight` ?

Reply:
We can't just add a bigger marginRight, because in one case we haven't a icon so : (padding of the card) 16px + 16px (marginRight) = 32px.
And in a second case we have the size of the icon who is added : 32px + 24px (icon's size) = 56px.

And if we try to have a dynamic margin like :     `<Box marginRight={isSelected ? 'md' : 'xl'}>` the content of the card move during the loading state (cf screenshot).


**_Solution adopted with you : keep the Box but with a spacing props in the marginRight_**  `<Box marginRight="lg" />`